crates/test-programs/Cargo.toml

```diff
@@ -20,3 +20,6 @@ futures = { workspace = true, default-features = false, features = ['alloc'] }
 url = { workspace = true }
 sha2 = "0.10.2"
 base64 = "0.21.0"
+# image and ndarray are used by nn_image_classification_onnx for image preprocessing.
+image = { version = "0.24.6", default-features = false, features = ["jpeg"] }
```

Comment:
We had avoided adding image and ndarray dependencies in favor of using the raw image used in the openvino test. What you did with the dog image is pretty close to what I had originally. @abrown, WDYT?

Reply:
Image processing and post-processing are copied from your classification-component-onnx example. If new dependencies are not allowed, we can do preprocessing offline, just put a RGB file here.

Reply:
Personally, I like showing the image prep / processing as it provides guidance to folks learning how to get started. In fact, it'd probably be helpful to add more comments / documentation to samples to explain what is happening. I think we, as practitioners, may be too accustomed to the transformations we regularly do to realize how odd they may appear to the naive observer.

Reply:
My original idea was to use an image as input for all backends, so we can cross check the results for correctness.

To avoid introducing extra dependencies, the image is replaced with processed tensor data (000000062808.rgb). I believe this is the same image for openvino backend ([tensorization for openvino](https://github.com/intel/openvino-rs/blob/7efb30d207e874d1dbf727bb52949a343d726567/crates/openvino/tests/fixtures/mobilenet/build.sh#L42)).